src/components/structures/UserSettings.js

```diff
@@ -284,7 +284,12 @@ module.exports = React.createClass({
         this.setState({ electron_settings: settings });
     },
 
-    _refreshMediaDevices: function() {
+    _refreshMediaDevices: function(stream) {
+        if (stream) {
+            // kill stream so that we don't leave it lingering around with webcam enabled etc
```

Comment:
Perhaps also explain that we don't need streams now, we just need device IDs to populate the audio and video input dropdowns.